.github/workflows/build.yml

```diff
@@ -215,19 +215,22 @@ jobs:
           ref: ${{ needs.set-tags.outputs.git_ref }}
       - name: Login to DockerHub
         # The if must stay on this step (because this job must not be skipped)
-        if: ${{ github.event.pull_request.head.repo.full_name == github.repository }}
+        if: |
```

Comment:
shouldn't it be if git_branch == master ?

Reply:
This is probably dangerous also.
We don't want another repository to be able to access our docker

Reply:
I rework the condition logic and added a comment to explain why